tests/specs/utils/http.spec.js

```diff
@@ -114,18 +114,22 @@ describe('HTTP', () => {
             });
         });
 
+        function randomIntBetween(min, max) {
+            return Math.floor(Math.random() * (max - min)) + min;
+        }
+
         describe('#handleStatus', () => {
-            for (let status = 200; status < 300; status++) {
+            [200, randomIntBetween(201, 299), 299].forEach((status) => {
```

Comment:
I'll get rid of the random tests, but I'm not convinced of the necessity of testing all status codes between 200 and 599. Apart from special cases like 202 and 204, they don't represent distinct behaviors in the code.

When you get down to it, the test is ensuring our correct use of `>=` and `<` operators and their operands, and defending against regressions that change these bounds. To sufficiently validate this, you can simply need to check the boundaries, not every value in between.

Instead of the random values, I'll use a few common standard HTTP status codes instead. What do you think?